specs/monitor_ceph_pool_utilization.adoc

```diff
@@ -0,0 +1,190 @@
+= Collect and process ceph pool utlization trending data
+
+The performance monitoring module if enabled in Tendrl, should collect and
+process the ceph pool utilization data in time series DB. This would help in
+creating trending charts in Tendrl UI which can help storage administrator to
+decide on requirement for additional storage in the system.
+
+== Problem description
+
+If ceph pool utilization trending is not available to administrator, they cannot
+easily decide how the pools are getting filled up and at what point in time more
+additional storage to be brought in and provisioned in the system.
+
+== Use Cases
+
+This PR addresses the trending charts use case in Tendrl for the ceph storage
+pools.
+
+== Proposed change
+
+* Introduce a collectd plugin which collects the ceph pool utilization data from
+underlying cluster and pushes it to time series DB at regular intervals.
+
+* Enhance the monitoring module to set required thresholds for the ceph pool
+utilization
```

Comment:
Also I think it would be better to mention the command or approach taken in the plugin to collect usage stats because, as you have mentioned below there are many techniques available to achieve this task we probably need to evaluate the flaws of each of them and make a final call. 1 problem that I have seen with ceph df with json format option is it provides only used size and % used but not total. I remember hearing that it'll be easier to add it on ceph side but we might need to raise an issue and track it if we go with that approach.